Inspicio/Controllers/ImagesController.cs

```diff
@@ -107,29 +108,51 @@ public class DataToBody
                 return NotFound();
             }
 
+            // Jack Lloyd 
+            // Fetch the image by the id pass in '/5'
             var image = await _context.Images.SingleOrDefaultAsync(m => m.ImageID == id);
             if (image == null)
             {
                 return NotFound();
             }
 
-            DataToBody DataToBody = new DataToBody();
-            DataToBody.Image = image;
+            // Jack Lloyd 
+            // Create a new FullReview object
+            // This will be passed into the view as the model.
+            FullReview FullReview = new FullReview();
+            FullReview.Image = image;
 
+            // Jack Lloyd 
+            // Fetch all the comments 
+            // This may not be the best way to do this...
             var allcomments = _context.Comments;
```

Comment:
If you're going with the "usual" C# naming style, this one should be `allComments` instead of `allcomments`